pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -96,9 +98,18 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 		}
 	}
 
-	err = validateIdentityUrl(doc.OpenShiftCluster, identityURL, isCreate)
-	if err != nil {
-		return nil, err
+	if isCreate {
+		// Persist identity URL and tenant ID only for managed/workload identity cluster create
+		// We don't support updating cluster managed identity after cluster creation
+		// TODO - use a common function to check if the cluster is a managed/workload identity cluster
```

Comment:
Is there a card or something to capture this for MIWI? (@kimorris27 maybe?)

Reply:
Yep, we've got https://issues.redhat.com/browse/ARO-7856 for this.